src/Plugin/OgDeleteOrphans/Cron.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\og\Plugin\OgDeleteOrphans;
+
+use Drupal\og\OgDeleteOrphansBase;
+
+/**
+ * Performs a cron deletion of orphans.
+ *
+ * @OgDeleteOrphans(
+ *  id = "cron",
+ *  label = @Translation("Cron", context = "OgDeleteOrphans"),
+ *  description = @Translation("The deletion is done in the background during cron. Best overall solution but requires cron to run regularly."),
+ *  weight = 3
+ * )
+ */
+class Cron extends OgDeleteOrphansBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function process() {
```

Comment:
I reworked it a bit to make it cleaner. The QueueWorker is now calling the process() method, which as an added bonus has deduplicated the code to delete an orphan. I've also updated the documentation to clarify it.

By the way it can be discussed whether or not we use a separate queue for the cron jobs or not. The way it works is that each QueueWorker is attached to a certain Queue. If there are items in the queue, then the queue worker WILL process them on every cron run, if the "Cron" plugin is selected or not.

Initially I was using the same queue, which has the benefit that if you select the Batch plugin and forget to press the button, then the orphans will still be cleaned up on cron over time.

In a next iteration I picked a different queue because I thought that people would find it strange that their orphans would be cleared up on cron if the option was not chosen.

Which solution would you prefer? We can use the same queue if we mention in the UI that orphans will always be cleaned up on cron.